### DISCUSSION TITLE
Feature Idea: Archived branches

### DISCUSSION THREAD

**Discussion by benstpierre** [community/community#48417] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

We have a growing number of branches that are "old".  We keep them around because they might be a failed experiment,  Something we cannot work on again for months/years more, or something we want to keep around but not actually incorporate.  

We typically have 10-15 branches that represent what we are actually working on.

It would be amazing it we had an "archive" button on the branch which would move it to a section in github of "archived" branches.  We could unarchive it if/when we care to but it would no longer clutter our branch list/workflow.  It would be great if archived branches did now show up when you fetch the list of branches from remote in your IDE.

Anybody else find this sort of feature compelling?

**Comment by spenserblack** [comment]
This isn't exactly what you're requesting, but, with branch protections, you can lock a branch to make it read-only.
